packages/react-cards/src/components/CardFooter/useCardFooterClasses.ts

```diff
@@ -1,13 +1,9 @@
-import { makeClasses } from '@fluentui/react-theme-provider/lib/compat/index';
+import { ax, makeStyles } from '@fluentui/react-make-styles';
```

Comment:
Should we move this to a `useCardFooterStyles.ts` file to follow the convention you're using with the rest of the changes?

Reply:
Yes 👍 _overlooked this file_